spec/integration/image-streamer/api300/plan_script/create_spec.rb

```diff
@@ -35,6 +35,27 @@
       expect(item['planType']).to eq(options[:planType])
       expect(item['content']).to eq(options[:content])
     end
+
+    it 'creates a plan script with custom attributes' do
+      options = {
+        description: 'Description of this plan script',
+        name: PLAN_SCRIPT2_NAME,
+        hpProvided: false,
+        planType: 'deploy',
+        content: 'esxcli system hostname set --domain "@DomainName@"'
+      }
+
+      item = klass.new($client_i3s_300, options)
+      expect { item.create! }.not_to raise_error
+      item.retrieve!
+      expect(item['uri']).to be
+      expect(item['name']).to eq(options[:name])
+      expect(item['description']).to eq(options[:description])
+      expect(item['hpProvided']).to be options[:hpProvided]
```

Comment:
I think so he just wants verify the presence of uri, because the uri is generated by oneview server. He does not know the value of uri.